.github/workflows/bors_ci.yml

```diff
@@ -0,0 +1,374 @@
+name: Bors CI 
+on:
+  push:
+    branches:
+      - '_bors_staging'
+      - '_bors_trying'
+    paths:
+      - '.github/workflows/ci.yml'
+      - 'discovery_engine/**'
+      - 'discovery_engine_core/**'
+      - 'discovery_engine_flutter/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+env:
+  DART_VERSION: '2.14.4'
+  DART_WORKSPACE: ${{ github.workspace }}/discovery_engine
+  RUST_WORKSPACE: ${{ github.workspace }}/discovery_engine_core
+  RUST_NIGHTLY: nightly-2021-09-09
+  RUSTFLAGS: "-D warnings"
+  RUSTDOCFLAGS: "-D warnings"
+
+
+jobs:
+  selection:
```

Comment:
Can't we call the other one like we do in `xayn_ai`?

Reply:
I'm not sure if you are refering to the name of the CI? I changed the name of them to match with the ones with `xayn_ai`

Reply:
Sorry I meant if we can call `ci.yml` and not duplicating it here.

Reply:
Ah ok. Yes, I can create workflow reuse.